data/oh/legislature/Thomas-Hall-7948afad-ec3c-4e4c-a073-305fb8cd1b21.yml

```diff
@@ -1,20 +0,0 @@
-id: ocd-person/7948afad-ec3c-4e4c-a073-305fb8cd1b21
-name: Thomas Hall
-given_name: Thomas
-family_name: Hall
-image: https://www.legislature.ohio.gov/Assets/HouseMembers/Headshots/Medium/2947.jpg
-party:
-- name: Republican
-roles:
-- type: lower
```

Comment:
@bfossen-ce this file should not be deleted. This person ended their time in district 53 on 01-01-2023 but began serving in house district 46 on the same date.